bin/create-env-secrets.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+echo "====="
+echo "Welcome to the environment secrets setter!"
+echo "====="
+echo "Please select the target environment for the secrets:"
+echo "IMPORTANT: Ensure you are authenticated with the"
+echo "AWS CLI with the appropriate account,"
+echo "and using the correct profile."
+
+PS3='Please enter your choice: '
+options=("UAT" "SIT" "PROD" "Quit")
+select opt in "${options[@]}"
+do
+    case $opt in
+        "UAT")
+            ENV="uat"
+            break
+            ;;
+        "SIT")
+            ENV="sit"
+            break
+            ;;
+        "PROD")
+            ENV="prod"
+            break
+            ;;
+        "Quit")
+            exit
+            ;;
+        *) echo "invalid option $REPLY";;
+    esac
+done
+echo "You have selected the $ENV environment!"
+
+secrets_description=(
+    "csdap-client-id::The CSDAP client id."
+    "csdap-client-password::The CSDAP client password."
+    "csdap-host-url::The CSDAP host url."
+)    
+
+echo ""
+echo "===="
+echo ""
+
+for secret in "${secrets_description[@]}"
+do
+    secret_name="${secret%%::*}"
+    secret_description="${secret##*::}"
+    secret_full_path="csdap-cumulus/$ENV/$secret_name"
+    echo "*** $secret_name ***"
+    echo "This is the created secret full path: $secret_full_path"
+    echo "Description: $secret_description"
+    echo "Please input the value for $secret_name:"
+    echo "(or enter Q to skip this secret.)"
+    echo ""
+
+    read secret_value
```

Comment:
Since we are entering secrets, do we want to use the `-s` option to avoid echoing the values to the terminal?  If so, then we might need to ask the user to re-enter the secret and check that both inputs are the same, although this is likely overkill since each secret is likely copied and pasted, not manually typed out.

Reply:
I like that! Added it